modules/stanford_soe_helper_homepage/README.md

```diff
@@ -0,0 +1,3 @@
+# Stanford SoE Helper for the homepage
```

Comment:
Just do it now. It is mostly copy pasta work and the ticket will never get prioritized.